? libbeagle/examples/beagle-info
? search/beagle-search.desktop.in
? tools/beagle-settings.desktop.in
Index: search/Tiles/MailMessage.cs
===================================================================
RCS file: /cvs/gnome/beagle/search/Tiles/MailMessage.cs,v
retrieving revision 1.23
diff -u -r1.23 MailMessage.cs
--- search/Tiles/MailMessage.cs	13 Sep 2006 06:19:36 -0000	1.23
+++ search/Tiles/MailMessage.cs	18 Nov 2006 15:34:43 -0000
@@ -133,12 +133,14 @@
 				p.Arguments = new string [2];
 				p.Arguments [0] = "evolution";
 				p.Arguments [1] = uri;
+#if ENABLE_THUNDERBIRD
 			} else if (client == "thunderbird") {
 				p = new SafeProcess ();
 				p.Arguments = new string [3];
 				p.Arguments [0] = Thunderbird.ExecutableName;
 				p.Arguments [1] = "-mail";
 				p.Arguments [2] = uri;
+#endif
 			}
 
 			return p;
